packages/test/stochastic-test-utils/src/performActions.ts

```diff
@@ -90,8 +93,8 @@ export async function performFuzzActionsAsync<
 		| { [K in TOperation["type"]]: AsyncReducer<Extract<TOperation, { type: K }>, TState> },
 	initialState: TState,
 	saveInfo?: SaveInfo,
+	savedInfo?: SavedInfo,
```

Comment:
Instead of adding code to this function (and the synchronous version of this function), we could just provide a helper function that can produce an `AsyncGenerator` given the file path to the saved operations. This way the caller of `performFuzzActionsAsync` can just pass that generator in.

Reply:
See the usage of `generatorFromArray` in `replayTestFromFailureFile` in packages\dds\task-manager\src\test\taskManager.fuzz.spec.ts